manifests/init.pp

```diff
@@ -66,16 +87,19 @@
 
     class { '::puppet_agent::prepare':
       package_file_name => $_package_file_name,
+      package_version   => $package_version,
     } ->
     class { '::puppet_agent::install':
       package_file_name => $_package_file_name,
+      package_version   => $package_version,
     }
 
     contain '::puppet_agent::prepare'
```

Comment:
I think MODULES-3118 implies this needs to happen even if the aio_agent_version and package_version match.